tests/jballerina-integration-test/src/test/resources/profiler/profiler_single_file.bal

```diff
@@ -60,7 +60,7 @@ public function main() {
     Person person = new("John", "Doe");
     test:assertEquals("John", person.firstName);
     test:assertEquals("John Doe", person.'\$init\$());
-    test:assertEquals("this is a user defined function", '\.\<init\>());
+    // test:assertEquals("this is a user defined function", '\.\<init\>());
```

Comment:
Similar, https://github.com/ballerina-platform/ballerina-lang/pull/41540/files#r1527823270

Reply:
Similar to above comment